airflow/providers/cncf/kubernetes/operators/kubernetes_pod.py

```diff
@@ -143,6 +151,12 @@ class KubernetesPodOperator(BaseOperator):
     :param priority_class_name: priority class name for the launched Pod
     :param termination_grace_period: Termination grace period if task killed in UI,
         defaults to kubernetes default
+    :param pod_runtime_info_envs: (Optional) A list of environment variables,
+        to be set in the container. Cannot be updated.
```

Comment:
```suggestion
    :param pod_runtime_info_envs: (Optional) A list of environment variables,
        to be set in the container.
    :param termination_grace_period: Termination grace period if task killed in UI,
        defaults to kubernetes default
```

Reordered, plus we don't need the 'Cannot be updated', as this doesn't make sense in the KPO context.